### DISCUSSION TITLE
Can't download on Paramount Plus

### DISCUSSION THREAD

**Discussion by szwanger** [aclap-dev/video-downloadhelper#762] [Help and Q&A]
### Your question

Bug details:

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "details": "SideDownload error: : [h264 @ 00000241afeb7e00] top block unavailable for requested intra mode\r\n[h264 @ 00000241afeb7e00] error while decoding MB 6 0, bytestream 44\r\n[h264 @ 00000241afee6f80] top block unavailable for requested intra mode -1\r\n[h264 @ 00000241afee6f80] error while decoding MB 1 0, bytestream 120\r\n[h264 @ 00000241b02dbb40] top block unavailable for requested intra mode -1\r\n[h264 @ 00000241b02dbb40] error while decoding MB 6 0, bytestream 123\r\n[h264 @ 00000241b033d500] top block unavailable for requested intra mode -1\r\n[h264 @ 00000241b033d500] error while decoding MB 1 0, bytestream 184\r\n[h264 @ 00000241b0345bc0] top block unavailable for requested intra mode -1\r\n[h264 @ 00000241b0345bc0] error while decoding MB 1 0, bytestream 71\r\n[h264 @ 00000241b0380e40] top block unavailable for requested intra mode -1\r\n[h264 @ 00000241b0380e40] error while decoding MB 5 0, bytestream 381\r\n[h264 @ 00000241b03e1ac0] top block unavailable for requested intra mode -1\r\n[h264 @ 00000241b03e1ac0] error while decoding MB 1 0, bytestream 452\r\n[eac3 @ 00000241b03ebe40] invalid bitstream id\r\n[eac3 @ 00000241b03ebe40] unable to determine channel mode\r\n[aac @ 00000241b0468100] channel element 3.4 is not allocated\r\n[out#0/mp4 @ 00000241b0ac5440] Error opening output F:\\Watch The Price Is Right Season 52 Episode 157 The Price is Righ.mp4: No such file or directory\r\nError opening output file F:\\Watch The Price Is Right Season 52 Episode 157 The Price is Righ.mp4.\r\nError opening output files: No such file or directory\r\n",
    "downloadable_id": "downloadable_4552770477000936",
    "error": "coapp_failure",
    "id": "coapp_failure_5d95a31f-f535-4642-ac62-8408261a4331"
  }
}

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "download_directory": "F:\\",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 7
}

**Comment by Mattmjs** [comment]
If they are a subscription service for tv, movies then it's likely they use DRM to protect the content. 
This can be verified quite easily if your using Firefox but since your using Chrome there is no clue what's protected and what isn't.
